devops/continuous_delivery/new_release_branch.rst

```diff
@@ -88,22 +88,33 @@ Change the database of the test customer in the created **release** branch:
 
       The commit will automatically be merged into master and needs to be reverted there.
 
-Create release branch in tocco-client repository
+New Client version change
 ------------------------------------------------
 Head over to the `tocco-client Repository`_ and create a new release branch based on the current master revision.
 Replace **${VERSION}** with the version number without any characters which aren’t numeric (e.g. 2.18 -> 218).
 
 .. _tocco-client Repository: https://github.com/tocco/tocco-client
 
-.. parsed-literal::
+.. code-block:: console
 
-   git checkout -b nice-releases/**${VERSION}** && git push
+        git checkout -b nice-releases/${VERSION} && git push
 
 Afterwards checkout master again and replace the nice version inside the file `nice-current-version`_ . 
 This change must be committed and pushed and a pull-request should be opened.
 
 .. _nice-current-version: https://github.com/tocco/tocco-client/blob/master/nice-current-version.txt
 
+The last step is to create a npm distribution tag for each package for the new nice version. Usually this could be achieved by 
+creating a new tag that point to the same version as the previous nice release tag.
+
+.. code-block:: console
+
+        npm dist-tag add tocco-entity-browser@${LATEST VERSION} nice229
```

Comment:
I'd replace `tocco-entity-browser` and `nice229` with placeholders too and add a real-life example below